portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,8 +32,6 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Since the |numComments| variable is being parsed from the request, what happens when it's out of range? What if the data store is empty, or you request more comments than the current max?

This is something that might be good to unit test later. :)